File: 02-tech/03-software-technologies/09-exam/C#/IssueTracker/Models/Issue.cs
namespace IssueTracker.Models
{
    using System.ComponentModel.DataAnnotations;

    public class Issue
    {
        public Issue() { }

        public Issue(string title, string content, int priority)
        {
            this.Title = title;
            this.Content = content;
            this.Priority = priority;
        }

        [Key]
        public int Id { get; set; }

        [Required(AllowEmptyStrings = false)]
        public string Title { get; set; }

        [Required(AllowEmptyStrings = false)]
        public string Content { get; set; }

        [Required]
        public int Priority { get; set; }
    }
}

File: 02-tech/03-software-technologies/09-exam/JavaScript/models/issue.js
const Sequelize = require('sequelize');

/**
 *
 * @param {Sequelize.Sequelize} sequelize
 * @returns { Sequelize.Model }
 */
module.exports = function (sequelize)
{

    let Issue = sequelize.define("Issue",
    {
        title:
        {
            type: Sequelize.TEXT,
            allowNull: false,
            // @ts-ignore
            required: true,
            validate:
            {
                notEmpty: true
            }
        },
        content:
        {
            type: Sequelize.TEXT,
            allowNull: false,
            required: true,
            validate:
            {
                notEmpty: true
            }
        },
        priority:
        {
            type: Sequelize.INTEGER,
            allowNull: false,
            required: true
        }
    },
    {
        timestamps: false
    });

    return Issue;
};